### DISCUSSION TITLE
WhiteBox with GEE

### DISCUSSION THREAD

**Discussion by Biku247** [opengeos/whitebox-python#38] [General]
Is there any way to integrate WhiteBox with Google Earth Engine for performing the Geomorphic, Hydrologic analysis within Google Earth Engine Environment? That would have been really great!

**Comment by giswqs** [comment]
Unfortunately, this is impossible. Just you like you cannot use ArcGIS with GEE. All objects in GEE are sever-side objects. You cannot use them with traditional GIS and remote sensing software unless you export GEE data to your local computer.
